### PR TITLE
Oidc creds 2066

### PR DESCRIPTION
**Pull request by irisked**

### CODE REVIEW

File: source/includes/_tenants.md [hunk 1]
```diff
@@ -2091,3 +2091,310 @@ This request is authorized for the users with the `tenants.oauth_clients.delete`
...
+  "data":{
+    "id": "5c80e6b9bb0d200011333d92",
+    "type": "openid-provider",
+    "attributes": {
+        "client_id": "c7e56633-5e88-4c97-8da9-f823432423",
+        "client_secret": "c7e56633-5e88-4c97-8da9-f823432423",
+        "issuer": "https://issuer.com",
+        "response_type": "code",
+        "config": {
+            "create_new_user_if_not_found": true
```
Comment (drobiazko) [inline source/includes/_tenants.md]: let's rename to: `"autoCreateUsers":true`

File: source/includes/_tenants.md [hunk 2]
```diff
@@ -2091,3 +2091,310 @@ This request is authorized for the users with the `tenants.oauth_clients.delete`
...
+    {
+      "id": "5c80e6b9bb0d200011333d92",
+      "type": "openid-provider",
+      "attributes": {
+          "client_id": "c7e56633-5e88-4c97-8da9-f823432423",
+          "client_secret": "c7e56633-5e88-4c97-8da9-f823432423",
+          "issuer": "https://issuer.com",
+          "response_type": "code",
+          "config": {
+              "create_new_user_if_not_found": true
```
Comment (drobiazko) [inline source/includes/_tenants.md]: Same as above

File: source/includes/_tenants.md [hunk 3]
```diff
@@ -2091,3 +2091,310 @@ This request is authorized for the users with the `tenants.oauth_clients.delete`
...
+  "data":{
+    "id": "5c80e6b9bb0d200011333d92",
+    "type": "openid-provider",
+    "attributes": {
+        "client_id": "c7e56633-5e88-4c97-8da9-f823432423",
+        "client_secret": "c7e56633-5e88-4c97-8da9-f823432423",
+        "issuer": "https://issuer.com",
+        "response_type": "code",
+        "config": {
+            "create_new_user_if_not_found": true
```
Comment (drobiazko) [inline source/includes/_tenants.md]: Same as above

File: source/includes/_tenants.md [hunk 4]
```diff
@@ -2091,3 +2091,310 @@ This request is authorized for the users with the `tenants.oauth_clients.delete`
...
+   -H 'Content-Type: application/json' -d '
+   {  
+     "data":{  
+       "type":"openid-provider,
+       "attributes":{
+         "issuer": "{ISSUER}"
+         "client_id":"{CLIENT_ID}",
+         "client_secret":"{CLIENT_SECRET}",
+         "config": {
+           "create_new_user_if_not_found": true
```
Comment (drobiazko) [inline source/includes/_tenants.md]: Same as above

File: source/includes/_tenants.md [hunk 5]
```diff
@@ -2091,3 +2091,310 @@ This request is authorized for the users with the `tenants.oauth_clients.delete`
...
+  "data":{
+    "id": "5c80e6b9bb0d200011333d92",
+    "type": "openid-provider",
+    "attributes": {
+        "client_id": "c7e56633-5e88-4c97-8da9-f823432423",
+        "client_secret": "c7e56633-5e88-4c97-8da9-f823432423",
+        "issuer": "https://issuer.com",
+        "response_type": "code",
+        "config": {
+            "create_new_user_if_not_found": true
```
Comment (drobiazko) [inline source/includes/_tenants.md]: Same as above

File: source/includes/_tenants.md [hunk 6]
```diff
@@ -2091,3 +2091,310 @@ This request is authorized for the users with the `tenants.oauth_clients.delete`
...
+### Payload Parameters
+
+| Parameter                                     | Required | Description                      |
+| --------------------------------------------- | -------- | -------------------------------- |
+| type                                          | yes      | A value should be "openid-provider"    |
+| attributes.issuer                             | no       | OpenID Provider issuer                 |
+| attributes.client_id                          | no       | OpenID Provider client ID              |
+| attributes.client_secret                      | no       | OpenID Provider client secret          |
+| attributes.response_type                      | no       | A value should be "code"             |
+| attributes.config.create_new_user_if_not_found | no       | A value should be true or false      |
```
Comment (drobiazko) [inline source/includes/_tenants.md]: Same as above

File: source/includes/_tenants.md [hunk 7]
```diff
@@ -2091,3 +2091,310 @@ This request is authorized for the users with the `tenants.oauth_clients.delete`
...
+### Payload Parameters
+
+| Parameter                                     | Required | Description                          |
+| --------------------------------------------- | -------- | ------------------------------------ |
+| type                                          | yes      | A value should be "openid-provider"    |
+| attributes.issuer                             | yes      | OpenID Provider issuer                 |
+| attributes.client_id                          | yes      | OpenID Provider client ID              |
+| attributes.client_secret                      | yes      | OpenID Provider client secret          |
+| attributes.response_type                      | no       | A value should be "code"             |
+| attributes.config.create_new_user_if_not_found | no       | A value should be true or false      |
```
Comment (drobiazko) [inline source/includes/_tenants.md]: Same as above

File: source/includes/_tenants.md (hunk 2 above)
Comment (irisked) [reply source/includes/_tenants.md]: done

File: source/includes/_tenants.md (hunk 1 above)
Comment (irisked) [reply source/includes/_tenants.md]: done

File: source/includes/_tenants.md (hunk 3 above)
Comment (irisked) [reply source/includes/_tenants.md]: done

File: source/includes/_tenants.md (hunk 4 above)
Comment (irisked) [reply source/includes/_tenants.md]: done

File: source/includes/_tenants.md (hunk 5 above)
Comment (irisked) [reply source/includes/_tenants.md]: done

File: source/includes/_tenants.md (hunk 6 above)
Comment (irisked) [reply source/includes/_tenants.md]: done

File: source/includes/_tenants.md (hunk 7 above)
Comment (irisked) [reply source/includes/_tenants.md]: done

**Pull request merged**